request-manager/graph/grapher.go

```diff
@@ -10,17 +10,20 @@ import (
 	"github.com/square/spincycle/v2/request-manager/spec"
 )
 
-// Grapher builds sequence graphs. A single node in a sequence graph (SG) represents
+// Grapher builds sequence graphs. A single node in a sequence graph represents
 // a node spec, and edges indicate dependencies between nodes in the spec. Each
 // sequence graphs describes a single sequence spec.
-// Sequence, conditional, and expandable sequence nodes are represented by one
-// node in a sequence graph, and are later resolved by the Resolver.
+// Sequence, conditional, and expandable sequence nodes correspond to a single
+// node in a sequence graph, and are later resolved into a request subgraph
+// consisting of only job nodes by the Resolver.
 type Grapher struct {
 	// User-provided
 	allSequences map[string]*spec.Sequence // All sequences read in from request specs
```

Comment:
Better name for this var: `seqSpecs` to make it clear at a glance that it's specs.